static/js/components/CommentList.jsx

```diff
@@ -285,23 +291,35 @@ const CommentList = ({
     }
     const spectrum = spectra[objID].find((spec) => spec.id === spectrumID);
     comments = spectrum?.comments;
+    comments?.forEach((comment) => {
+      comment.resourceType = associatedResourceType;
+    });
   } else if (associatedResourceType === "gcn_event") {
     if (gcnEventID === null) {
       throw new Error("Must specify a gcnEventID for comments on gcnEvent");
     }
     comments = gcnEvent.comments;
+    comments?.forEach((comment) => {
+      comment.resourceType = associatedResourceType;
+    });
   } else if (associatedResourceType === "shift") {
     if (shift_id === null) {
       throw new Error("Must specify a shiftID for comments on shift");
     }
     comments = currentShift?.comments;
+    comments?.forEach((comment) => {
+      comment.resourceType = associatedResourceType;
+    });
```

Comment:
Would it be easier to add this info in the handler like we did for Annotations?

Reply:
Good idea.

Reply:
Yes this was a good idea.